databook/background/Mouse-visual-system.md

```diff
@@ -39,11 +39,11 @@ organized in columns, the rodent visual cortex has a "salt and pepper"
 organization, where the orientation selectivity of neighboring neurons are
 randomly organized. {cite:p}`ohki`
 
-Beyond {term}`V1` there are additional higher visual areas (or {term}`HVA`s)
-that receive input from V1 and are connected in a hierarchical manner. Higher
-visual areas each contain a distinct map of {term}`Retinotopy`, i.e. each visual
-area has its own map of visual space. The borders between higher visual areas
-can be identified based on reversals in the retinotopic map at area boundaries.
+Beyond {term}`V1` there are additional higher visual areas ({term}`HVA`s)
+which receive input from V1 and are connected in a hierarchical manner. HVAs
+each contain a distinct map of {term}`Retinotopy`, i.e. each visual
+area contains a complete map of the visual space. The borders between higher visual 
```

Comment:
I'd avoid saying "complete" because they have different biases in retinotopic coverage.

Reply:
removed word "complete"